<?xml version="1.0"?>
<launch>
    <param name="max_iter" value="30"/>
    <param name="step_size" value="0.1"/>
    <param name="ndt_res" value="5.0"/>
    <param name="trans_eps" value="0.01"/>
    <param name="voxel_leaf_size" value="2.0"/>
    
    <node pkg="front_odometry" type="front_odometry" name="front_odometry" output="screen"></node>

    <node pkg="rviz" type="rviz" name="rviz" required="true"></node>
</launch>



